test/examples.cpp

```diff
@@ -181,17 +181,17 @@ struct ArgsAsList
       /*<< Use a `reverse_fold<>` transform to iterate over the children
       of this node in reverse order, building a fusion list from back to
       front. >>*/
-      , proto::reverse_fold<
+      , proto::fold<
```

Comment:
Change the comment to agree with the implementation, please.

Reply:
Done